Guided Triage - Alerts.ipynb

```diff
@@ -44,7 +59,7 @@
     "# Install required packages for this Notebook\n",
     "!pip install msticpy --upgrade --user\n",
     "!pip install python-whois --user --upgrade\n",
-    "!pip install git+\"https://github.com/tqdm/tqdm\" --upgrade --user\n",
+    "!pip install tqdm --upgrade --user\n",
```

Comment:
So this was deliberate - the tqdm project is current going through a change of ownership meaning current releases are only going out via GitHub and not PyPI. I was going to update this notebook once they are publishing to PyPI again.

Reply:
Yeah, the version is identical in PyPi version and the github master. I didn't check the code though. BTW I love the pandas tqdm progress. That is just a fabulous find.